### ISSUE TITLE
Dialog is not correctly created

### ISSUE THREAD

**Issue by Felix2048** [psychopy/psychopy#3089]
I'm running PsychoPy on win10 and here's my code:
```python
dlg = gui.DlgFromDict(dictionary=expInfo, sortKeys=False, title=expName)
```
Then the dialog failed to create:
```bash
>>> dlg = gui.DlgFromDict(dictionary=expInfo, sortKeys=False, title=expName)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Anaconda3\envs\psychopy\lib\site-packages\psychopy\gui\wxgui.py", line 325, in __init__
    self.addField(field, self.dictionary[field], tip=tooltip)
  File "C:\Anaconda3\envs\psychopy\lib\site-packages\psychopy\gui\wxgui.py", line 330, in show
    def show(self):
  File "C:\Anaconda3\envs\psychopy\lib\site-packages\psychopy\gui\wxgui.py", line 233, in show
    """
  File "C:\Anaconda3\envs\psychopy\lib\site-packages\psychopy\gui\wxgui.py", line 192, in display
    self.sizer.Add(buttons, 1,
wx._core.wxAssertionError: C++ assertion "!(flags & wxALIGN_BOTTOM)" failed at ..\..\src\common\sizer.cpp(2066) in wxBoxSizer::DoInsert(): Vertical alignment flags are ignored in vertical sizers
```

So, I changed `wxgui.py` in `gui` package, line 191 to 192:
```python
        self.sizer.Add(buttons, 1,
                       flag=wx.ALIGN_RIGHT | wx.ALIGN_BOTTOM, border=5)
```
Into:
```python
        self.sizer.Add(buttons, 1,
                       flag=wx.ALIGN_RIGHT, border=5)
```
And, everything seems fine now.

**Comment by peircej** [comment]
Thanks Felix. Do you wanna send a pull request for this (commit should have prefix `BF:`) so that you get credit with the fix?

**Comment by mdcutone** [comment]
Those flags are mutually exclusive, surprised it hasn't been caught sooner. Must be something newer versions of wx strictly enforce.

**Comment by peircej** [comment]
Yes, I think there are a lot of aspects in wx where mistakes are now being trapped and reported rather than ignored (which would have been giving undefined behaviour - like in this case it's undefined which of the two flags was used)

**Comment by peircej** [comment]
Fixed in #3121 
Thanks again for picking it up Felix
